src/Core/BaseSettings.h

```diff
@@ -62,6 +67,8 @@ class BaseSettings : public TTraits::Data
 
     /// Resets all the settings to their default values.
     void resetToDefault();
+    /// Resets specified setting to its default value.
+    void resetToDefault(const std::string_view & name);
```

Comment:
```suggestion
    void resetToDefault(const std::string_view name);
```

Reply:
Why?

Reply:
it's an anti-pattern with string_view.
The point of it is to cheaply send string arguments around while eliminating extra indirection.

E.g.
To get to a value of `const std::string &` you have `ref` -> `const char *` -> `char`
For `const & std::string_view` it's the same while for `std::string_view` you only have `const char *` -> `char`.
Also, `string_view` contains only pointer and size, making it good for copying.

The same thing can be achieved with `std::span` for collections.